code/game/atoms_movable.dm

```diff
@@ -64,6 +64,15 @@
 	/// Whether this atom should have its dir automatically changed when it moves. Setting this to FALSE allows for things such as directional windows to retain dir on moving without snowflake code all of the place.
 	var/set_dir_on_move = TRUE
 
+	/**
+	 * an associative lazylist of relevant nested contents by "channel", the list is of the form: list(channel = list(important nested contents of that type))
+	 * each channel has a specific purpose and is meant to replace potentially expensive nested contents iteration
+	 * do NOT add channels to this for little reason as it can add considerable memory usage.
+	 */
```

Comment:
```suggestion
	/**
	* an associative lazylist of relevant nested contents by "channel", the list is of the form: list(channel = list(important nested contents of that type))
	* each channel has a specific purpose and is meant to replace potentially expensive nested contents iteration
	* do NOT add channels to this for little reason as it can add considerable memory usage.
	*/
```